iree/compiler/Conversion/LinalgToSPIRV/KernelDispatchUtils.cpp

```diff
@@ -708,6 +752,27 @@ Optional<LaunchConfig> initGPULaunchConfig(
 #undef DISPATCH
   }
 
+  if (!rootOperation) {
+    for (linalg::LinalgOp linalgOp : linalgOps) {
+      if (auto op = dyn_cast<linalg::GenericOp>(linalgOp.getOperation())) {
+        if (getNumOuterParallelLoops(linalgOp) == 0 ||
```

Comment:
When we drop the old path, this will have to go through the new path.

Reply:
we still need this even in the new path right?

Reply:
Sorry, bad comment. I was saying that this is basically saying dont do anything with 0-rank operations. But those need to be handled as well (i.e. lowered to loops albeit no-op) at some point. I am guessing this isnt tested on the new path. 
One thing to do here is just test what happens if a `linalg.generic` is just  0-rank operations.

Reply:
Good point, I can add a test for that.